Configuration/Skimming/python/PDWG_EXOCSCCluster_cff.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi
+CSCClusterTrigger = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone()
+CSCClusterTrigger.TriggerResultsTag = cms.InputTag( "TriggerResults", "", "HLT" )
+CSCClusterTrigger.HLTPaths = cms.vstring(
+    ["*CscCluster*","*L1CSCShower*"]
+)
+CSCClusterTrigger.throw = False
+CSCClusterTrigger.andOr = True
```

Comment:
Not for this PR, but for a possible cleaning in another PR:
```suggestion
CSCClusterTrigger = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
   TriggerResultsTag = ( "TriggerResults", "", "HLT" ),
   HLTPaths = ["*CscCluster*","*L1CSCShower*"],
   throw = False,
   andOr = True
)
```